server/src/domain/search/dto/search-suggestion.dto.ts

```diff
@@ -0,0 +1,40 @@
+import { PeopleResponseDto, PersonResponseDto } from '@app/domain';
+import { toBoolean } from '@app/domain/domain.util';
+import { ApiProperty } from '@nestjs/swagger';
+import { Transform } from 'class-transformer';
+import { IsArray, IsBoolean, IsEnum, IsNotEmpty, IsOptional, IsString } from 'class-validator';
+
+export enum SearchSuggestionType {
+  Country = 'country',
+  State = 'state',
+  City = 'city',
+  CameraMake = 'camera-make',
+  CameraModel = 'camera-model',
+}
+
+export class SearchSuggestionRequestDto {
+  @IsEnum(SearchSuggestionType)
+  @IsNotEmpty()
+  @ApiProperty({ enumName: 'SearchSuggestionType', enum: SearchSuggestionType })
+  type!: SearchSuggestionType;
+
+  @IsString()
+  @IsOptional()
+  country?: string;
+
+  @IsString()
+  @IsOptional()
+  state?: string;
+
+  @IsString()
+  @IsOptional()
+  make?: string;
+
+  @IsString()
+  @IsOptional()
+  model?: string;
+}
+
+export class SearchSuggestionResponseDto {
+  data?: string[];
+}
```

Comment:
I'm not sure if we even need this. I believe we can just return a string[] directly.